lms/static/styles/ui-playground/ui-playground.scss

```diff
@@ -13,3 +13,12 @@
 .content {
   display: block;
 }
+
+// The pattern library, by design, won't muck around with default element styling
+// which includes typography. In LMS' case, though, the default font size
+// is so tiny that it makes text in the pattern library unreadable.
+.LibraryExample__content,
```

Comment:
Because content within `LibraryExample` components is rendered within container elements, these styles won't "leak" into demos/examples. Unfortunately the same is not true for `LibraryPattern`, ergo the `.LMSLibrary__content` style which allows me to apply this surgically in a few places. I'm noting this for follow-up in Pattern-Library source.